go.sum

```diff
@@ -1317,6 +1317,7 @@ github.com/yuin/goldmark v1.1.27/go.mod h1:3hX8gzYuyVAZsxl0MRgGTJEmQBFcNTphYh9de
 github.com/yuin/goldmark v1.1.32/go.mod h1:3hX8gzYuyVAZsxl0MRgGTJEmQBFcNTphYh9decYSb74=
```

Comment:
Does the _Check Go Modules_ action pass without these changes to `go.mod` and `go.sum`?

Reply:
I'm not sure, but it also passes _with_ those changes. If it passes under both conditions there must be something wrong with the check, right? Surely it shouldn't be able to pass in both scenarios

Reply:
ec4e9b44eb upgrades `golang.org/x/tools` to `v0.1.8` and commits the corresponding `go.sum`. I also get those `go.sum` changes if I run `go mod tidy` after upgrading `golang.org/x/tools` to `v0.1.8`. So the check is not incorrect, just the https://github.com/apache/trafficcontrol/blob/ec4e9b44eb326b4f7d9a436cd7ec1300aa516b9a/go.mod#L78 line.

Reply:
That change is inserted by `go mod tidy`, though. I did not manually alter either `go.mod` or `go.sum`. If the check (and all other tests) pass after the change, shouldn't it stay in? The master branch also updates those files whenever I build a Go component, so the underlying issue isn't introduced in this PR.

Reply:
> If you download this changeset and run `go mod tidy`, does it change anything for you?

no